### DISCUSSION TITLE
How to upgrade remix version

### DISCUSSION THREAD

**Discussion by sumanta-ghosh** [remix-run/remix#5768] [Q&A]
At present, I am utilizing version 1.1.1 of Remix and wish to upgrade to the most recent version. However, I am unable to locate any relevant documentation on the subject. 

Could someone please provide assistance with this matter?

**Comment by janhesters** [comment]
Just bump your files in your `package.json` to the version you desire.

If you want to install the latest version, you can do so by installing each package with `@latest`, e.g.

```bash
npm i @remix-run/react@latest
```

**Reply by sumanta-ghosh** [reply]
but there are a lot of other changes in imports and a lot of different things are also deprecated in the latest version. Is there any proper document where everything is mentioned?
